Localization/fr-FR_Mods.SpelunkerFilter.hjson

```diff
@@ -1,94 +1,94 @@
 Configs: {
 	SFConfig: {
-		// DisplayName: Config
-		// DefaultFilterTooltip: Disable this to prevent the {$ItemName.SpelunkerPotion} from highlighting it
+		DisplayName: Configuration
+		DefaultFilterTooltip: Désactiver ceci pour empêcher la {$ItemName.SpelunkerPotion} de mettre ce bloc en surbrillance
 
 		Headers: {
-			// CustomFilter: Custom Filter
-			// PresetFilterGroupToggles: Preset Filter - Group Toggles for Vanilla Tiles
-			// PresetFilter: Preset Filter - Vanilla Tiles
+			CustomFilter: Filtre Personnalisé
+			PresetFilterGroupToggles: Filtre Préréglé - Groupe d'options pour les Blocs Vanilla
+			PresetFilter: Filtre Préréglé - Blocs Vanilla
 		}
 
 		ApplyToMetalDetector: {
-			// Label: Apply To Metal Detector
-			// Tooltip: If enabled, applies the same filter settings to the {$ItemName.MetalDetector}
+			Label: S'applique au {$ItemName.MetalDetector}
+			Tooltip: Si activé, applique le même effet au {$ItemName.MetalDetector}
 		}
 
 		RemoveSparklingDust: {
-			// Label: Remove Sparkling Dust
-			// Tooltip: If enabled, removes the sparkling dust particles that appear over highlighted tiles. May improve performance
+			Label: Retire l'effet de Scintillement
+			Tooltip: Si activé, retire les particules scintillantes qui apparaissent sur les blocs en surbrillance. Pourrait améliorer les performances.
```

Comment:
Si activé, retire les particules scintillantes qui apparaissent sur les blocs en surbrillance. Peut améliorer les performances.

Reply:
It's better since the entire sentence is in present tense.